scripts/integration-test.sh

```diff
@@ -3,7 +3,9 @@ INTEGRATION_TEST=$1
 SRC_PATH=$2
 GATSBY_PATH="${TRAVIS_BUILD_DIR:-../../}" # set to third arg if defined, otherwise use ../../
 
-if [[ "$INTEGRATION_TEST" = true ]]; then
+is_pull_request='^[0-9]+$'
+
+if [[ "$INTEGRATION_TEST" = true && "$TRAVIS_PULL_REQUEST" =~ $is_pull_request ]]; then
```

Comment:
If anyone knows a better way to compare here, I'm all 👂s

`$TRAVIS_PULL_REQUEST` is either false, if not a PR, or a number. This detects the latter case.